alkira/resource_alkira_service_f5_lb_helper.go

```diff
@@ -193,7 +190,6 @@ func setF5Instances(d *schema.ResourceData, ins []alkira.F5Instance) []map[strin
 			"registration_credential_id": in.RegistrationCredentialId,
 			"credential_id":              in.CredentialId,
 			"version":                    in.Version,
-			"deployment_option":          in.Deployment.Option,
```

Comment:
We should have a preset value for this? Or we could just leave this out? Backend will handle it?

Reply:
No, TPS has the default value now, so I'm avoiding a hardcoded value here.